### PR TITLE
textlintとreviewdogの動作確認

### PR DESCRIPTION
**Pull request by budougumi0617**

### CODE REVIEW

File: articles/budougumi0617.re [hunk 1]
```diff
@@ -0,0 +1,14 @@
+= 動作確認用
+
+== はじめに
```
Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
はじめ => 始め (prh)

File: articles/budougumi0617.re [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+freee株式会社でバックエンドエンジニアをしている@budougumi0617@<fn>{bd617_twitter}です。
```
Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
株式会社 => (株) (prh)

File: articles/budougumi0617.re [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+//footnote[bd617_twitter][@<href>{https://twitter.com/budougumi0617}]
+
+=== 本章の目的・ゴール
+このファイルはreviewdogの動作確認用です！
```
Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)

File: articles/budougumi0617.re [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+きっと長い文書の中で出来るだけの「の」を使ったりすればreviewdogの設定で呼ばれているtexlintのワーニングが呼び出されるはず！！！！
```
Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
出来る => できる (prh)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
"！" が連続して2回使われています。 (ja-technical-writing/ja-no-successive-word)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
"！" が連続して2回使われています。 (ja-technical-writing/ja-no-successive-word)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Disallow to use "！". (ja-technical-writing/no-exclamation-question-mark)

Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
"！" が連続して2回使われています。 (ja-technical-writing/ja-no-successive-word)

File: articles/budougumi0617.re [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+gitHubとかそんな感じで正式名称と違うjqueryとか書けばlintに引っかかるはず
```
Comment (budougumi0617) [inline articles/budougumi0617.re]: **[textlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
文末が"。"で終わっていません。 (ja-technical-writing/ja-no-mixed-period)

**Comment by MakoTano** [comment]
reviewdogのtextlintいいやつですね笑

**Comment by budougumi0617** [comment]
revewdogが私のトークンでコメントしてるので私が口うるさいおっさんになってますね笑

**Comment by budougumi0617** [comment]
ひとまずlintの設定だけ完了。

**Pull request merged**